src/reducers/CommentReducer.js

```diff
@@ -1,23 +1,44 @@
-import { CREATE_COMMENT, DELETE_COMMENT } from '../actions/Actions';
+import { CREATE_COMMENT, DELETE_COMMENT, DELETE_POST_COMMENT } from '../actions/Actions';
 
-export default function commentReducer(state = {}, action){
+export const initialState = {
+  comment: []
```

Comment:
I think you'll need the post index to tie this together so that you only display comments that make a post.